build/system.gyp

```diff
@@ -35,6 +35,25 @@
        ],
       ],
     },
+    {
+      'target_name': 'libnotify',
+      'type': 'none',
+      'conditions': [
+        ['building_crosswalk_bin==1', {
+          'defines': ['USE_LIBNOTIFY=1'],
+          'cflags': [
+            '<!@(pkg-config --cflags libnotify)',
+            '<!@(pkg-config --cflags gdk-pixbuf-2.0)',
```

Comment:
Why gdk-pixbuf-2.0 is required for the libnotify target, I do not see those two modules has dependency in nature. If your work requires both of them, then create two separate target make more sense to me.